packages/jsii-pacmak/test/diff-test.sh

```diff
@@ -29,11 +29,6 @@ function assert-generator() {
         rsync -a ${original_expected}/ ${expected}/
     fi
 
-    if [[ -d ${original_expected}/java/target ]]; then
```

Comment:
This helps me a lot. Please don't remove it?

Reply:
What problem does it solve? I had no problem after removing this... I'm curious what I'm missing!

It did however cause me **MUCH** pain, because my IDE just auto-touches stuff, and that made my iteration "edit code - close IDE - clean up files - test - open IDE - edit code - ...". This is painful.

Reply:
Yes, but for me NOT doing this check just leads to a more cryptic error downstream (you will see an error about the diff failing because one dir has target, .settings, .project).

I added a test explicitly to tell you WHY that was happening, because it was confusing to me.

If your IDE compiles the expectation dir, the test set will fail, regardless of whether this test is here or not.